katy-nike-collection.php

```diff
@@ -2,10 +2,58 @@
 /**
  * retrieves a DB connection to the collection of shoes DB
  */
-$db = new PDO('mysql:host=db; dbname=katy-collection', 'root', 'password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-return $db;
-    
+
+function getDb() {
+    $db = new PDO('mysql:host=db; dbname=katy-collection', 'root', 'password');
```

Comment:
I would advise putting this in a separate file and use gitignore so you don't upload your password to github